index.js

```diff
@@ -0,0 +1,187 @@
+const tripsURL = 'https://trektravel.herokuapp.com/trips';
+
+let getTrips = function getTrips() {
+
+  const successCallback = function(response) {
+    console.log('success!');
+    console.log(response);
+    console.log(response[0]);
+
+    let back = '<button class="button white-text float-right">Go back</button>';
+
+    let continents = ['All'];
+    for (let trip of response) {
+      if (!continents.includes(trip.continent)) {
+        continents.push(trip.continent);
+      }
+    }
+
+    let tableHead = '<tr>';
+    for (let attr in response[0]) {
+      // tableHead += '<th>' + attr.toUpperCase() + '</th>';
+      if (attr === 'continent') {
+        tableHead += '<th><select>'
+        for (let continent of continents) {
+          tableHead += '<option>' + continent + '</option>';
+        }
+        tableHead += '</select></th>'
+      } else {
+        tableHead += '<th>' + attr.toUpperCase() + '</th>';
+      }
+    }
+    tableHead += '</tr>'
+
+    let tableRow = '';
+    for (let trip of response) {
+      tableRow += `<tr data-id=${trip.id}>`;
+      for (let attr in trip) {
+        tableRow += '<td>' + trip[attr] + '</td>';
+      }
+      tableRow += '</tr>';
+    }
+    $('.back').append(back);
+    $('.trips table').append(tableHead);
+    $('.trips table').append(tableRow);
+
+    $('#trips').toggle();
+  } // end of successCallback
+
+  const failureCallback = function(response) {
+    let generatedHTML = '<p>Something went wrong! Please try again!</p>';
+
+    $('.trips').html(generatedHTML);
+  }; // end of failureCallback
+
+$.get(tripsURL, successCallback);
+
+
+} // end of getTrips
+
+let getTripBuilder = function(id) {
+  let getTrip = function getTrip() {
+    const successCallback = function(response) {
```

Comment:
This organization is a little wild - you're defining a function inside a function inside a function here. I think it should be possible to simplify this a bit.

In particular, the name of the outer function implies that you're building a closure here, but that's not the case. Instead of `return`ing `getTrip`, you invoke it at the bottom of the function.